### ISSUE TITLE
Term Customizer : cannot add a constraint for the landing page of a process group

### ISSUE THREAD

**Issue by marta-platoniq** [mainio/decidim-module-term_customizer#75]
**Describe the bug**
I want to customize some terms only in the landing page of a process group, but I cannot select the process group in the constraint filter of the term customizer module.

**To Reproduce**

1. Create a process group
2. Go to Term Customizer > New translation set
3. Check the constraint filter, can you find and restrict by the process group you have created?

**Expected behavior**
Somewhere in the filter, the process groups should appear.

**Screenshots**
![image](https://user-images.githubusercontent.com/72607737/125413071-d46766d3-ab2c-470c-b6eb-a83fb13adf3b.png)

**Extra data**
Decidim Version: 0.24

**Comment by ahukkanen** [comment]
Thanks for the feature suggestion @marta-platoniq !

We are no longer taking advantage of the process groups, so right now we don't have internally the need for such feature.

We would happily accept a pull request with relevant added test cases for this feature!

**Comment by eliegaboriau** [comment]
Hello @ahukkanen 
Have you seen [these tech specs](https://github.com/puzzle/decidim-zuerich/issues/327#issuecomment-1260868749] from puzzle ?
Do you see a simpler way to do this ? Or is it the right way to contribute for this issue ?
Thanks a lot

**Comment by ahukkanen** [comment]
@marta-platoniq @paulinebessoles @eliegaboriau @carlobeltrame 

Sorry for pinging you all but after having a review of #94, could you also share some of the actual use cases that you would be using this feature for? For us to understand the problem scope a bit better.

And by this I mean which of the strings you want to modify for the participatory groups because looking at least the content blocks on those pages, they seem to have already strings specific to that context only, even if you don't constrain the translation set only to participatory process groups.

I understand the usefulness of this kind of feature in general but I'm not perfectly satisfied with the approach taken at #94 to solve this issue. If we can understand the problem better, it makes it also easier to make a judgement if it is worth the effort.

**Comment by carlobeltrame** [comment]
@larsUE do you want to explain why this is needed in Zurich?

**Comment by larsUE** [comment]
Yes, of course. Our process groups in Zürich are used not only spatially, but also for organisations (not in the decidim sense, but e.g. for different government offices). We plan to use the process group main page as a sort of landing page for the participatory processes of these offices. That also means that depending on the office, some terms need to be adjusted, not just within the processes, but also on process group level (headings, filters, metadata boxes etc.).

Use case: We have a new municipal organisation bringing together all efforts in combatting climate change in Zürich. They develop a concept for the platform and decide that they will conduct various processes to gather ideas. However, it is really important for them from a communicative perspective that these are called ideas, not proposals. Changing that in a process is easy using Term Customizer, but we would also like to change the headings of the proposal section on the process group landing page.

**Comment by paulinebessoles** [comment]
Hi @ahukkanen ! Thank you for your feedback. I agree with @larsUE, we have the same type of demands by clients using groups with different semantics. We don't have specific translations for statistics semantics in groups content blocks for example, so a constraint is useful for us there. 

Can @eliegaboriau move forward and update his PR by integrating your review, or do you prefer another implementation?

**Comment by ahukkanen** [comment]
@paulinebessoles @eliegaboriau I think continuing the current PR is fine.
